joml-geometry/src/main/java/org/terasology/joml/geom/AABBf.java

```diff
@@ -262,6 +283,74 @@ public Vector3d center(Vector3d dest) {
         return dest.set(minX + ((maxX - minX) / 2.0), minY + ((maxY - minY) / 2.0), minZ + ((maxZ - minZ) / 2.0));
     }
 
+    @Override
+    public AABBf expand(float dx, float dy, float dz, AABBf dest) {
```

Comment:
The docstring tells 
```      
* @throws IllegalArgumentException if extending this region would result in any non-positive dimension 
``` 
but this is actually not the case for this class. Either remove that invariant from the docs or add preconditions as for BlockRegion

Reply:
And, as stated above, let's not implement the same thing multiple times.

Reply:
I probably got that from BlockRegion. umm, do we want to throw an exception.